src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -1064,7 +1066,7 @@ internal IEnumerable<SecurityKey> GetContentEncryptionKeys(JsonWebToken jwtToken
             if (unwrappedKeys.Count > 0 && exceptionStrings.Length == 0)
                 return unwrappedKeys;
             else
-                throw LogHelper.LogExceptionMessage(new SecurityTokenKeyWrapException(LogHelper.FormatInvariant(TokenLogMessages.IDX10618, keysAttempted, exceptionStrings, jwtToken)));
+                throw LogHelper.LogExceptionMessage(new SecurityTokenKeyWrapException(LogHelper.FormatInvariant(TokenLogMessages.IDX10618, keysAttempted, exceptionStrings, jwtToken.ToString())));
```

Comment:
Does this change do anything? I believe ToString is called by default anyways in string.Format which this eventually calls.

Reply:
That's correct. LogHelper.FormatInvariant internally invokes string.Format where ToString is called by default

Reply:
Given that this has no change on the output, and that changing the type of the parameter from JsonWebToken to string will prevent the LogHelper class to be able to handle this type in specific like it does for NonPII or Exception, I would not change it.

Reply:
Following the same train of thought, this change could be implemented entirely on the LogHelper.FormatInvariant by checking for the type of the object to be SecurityToken and if it is sanitize the output in place.